DESCRIPTION

```diff
@@ -24,7 +30,13 @@ Imports:
     ggplot2,
     jsonlite,
     anytime,
-    curl
+    curl,
+    crayon,
+    dplyr,
+    magrittr,
+    purrr,
+    stringr,
+    tibble
```

Comment:
I would say no to this but I am not the maintainer.

Reply:
Due to the size of import?